packages/react/src/components/Lightbox/README.md

```diff
@@ -0,0 +1,104 @@
+# Lightbox
+
+> The Lightbox component is to be utilized within IBM.com.
+
+## Getting started
+
+Here's a quick example to get you started.
+
+```scss
+// yourapplication.scss
+@import '@carbon/type/scss/font-face/mono';
+@import '@carbon/type/scss/font-face/sans';
+@include carbon--font-face-mono();
+@include carbon--font-face-sans();
+```
+
+> 💡 Only import font's once per usage
+
+```javascript
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { LightBox } from '@carbon/ibmdotcom-react';
+import '@carbon/ibmdotcom-styles/scss/components/lightbox/_lightbox.scss';
+import 'yourapplication.scss';
+
+function App() {
+  return;
+  <LightBox
+    title='Curabitur malesuada varius mi eu posuere'
+    copy='Lorem ipsum dolor sit amet, consectetur adipiscing Aenean et ultricies est.'
+    image={
+        uri: {
+          sm: 'https://via.placeholder.com/640x320',
+          md: 'https://via.placeholder.com/768x384',
+          lg: 'https://via.placeholder.com/1024x512',
+        },
+        alt: 'Placeholder Image',
+      }
+    open={true}
+    onClose={() => alert('<<< model closed >>>')}
+  />;
+}
```

Comment:
Looks like it's not returning the actual LightBox in the example

Reply:
Right, and are any of these props required? How about default props?

Reply:
Right, seems to be a closing `return;` Can you see if you can reconcile this against any of the other READMEs? @annawen1 did a refactor of the READMEs a little while ago.

Reply:
updated with [this commit](https://github.com/carbon-design-system/ibm-dotcom-library/pull/882/commits/afbd414592d1948f0861ff4866931e632f6389c6)